developer-docs-site/docs/nodes/staking.md

```diff
@@ -0,0 +1,236 @@
+:::
+
+- When your lockup period expires, you can choose not to extend it and instead call the following sequence of functions to leave the validator set:
+    - Call `Stake::unlock` to unlock your stake amount.
+    - Call `Stake::withdraw` to withdraw your staked amount at the next epoch, and finally
+    - Call `Stake::leave_validator_set` to be removed from the validator set.
+- When your lockup period is not expired, you can leave the validator set. However, in this case you cannot unlock or withdraw your stake.
+- Even when your lockup period has not expired, if, at the start of an epoch, your staked amount falls below the minimum required stake, then you will be automatically removed from the validator set at the start of the same epoch.
+  - In the current version of the staking on the Aptos blockchain, there is no possibility of your stake dropping below the required minimum before the lockup period expires. **This will change when Aptos implements slashing, i.e., penalty for malicious validator behavior.** 
+    
+
+:::tip Leaving the validator set
+For step-by-step instructions on how to leave the validator set, see: [Leaving Validator Set](https://aptos.dev/nodes/ait/connect-to-testnet#leaving-validator-set).
+:::
+
+## Rejoining the validator set
+
+When you leave a validator set, you can rejoin by depositing the minimum required stake amount for the minimum required lockup period.
+
+## How a custodian can stake on Aptos
+
+The Aptos staking module defines a capability that represents ownership. See [https://github.com/aptos-labs/aptos-core/blob/0daade4f734d1ba29a896b00d7ddde2249e87970/aptos-move/framework/aptos-framework/sources/configs/stake.move#L85](https://github.com/aptos-labs/aptos-core/blob/0daade4f734d1ba29a896b00d7ddde2249e87970/aptos-move/framework/aptos-framework/sources/configs/stake.move#L85).
+
+This `OwnerCapability` resource can be used to control the node operator (i.e., who runs the validator node) and the associated stake pool.
+
+Using this owner-operator model, a custodian can stake on the Aptos blockchain. This allows delegations and staking services to be built as the owner can provide funds to the validator.
+
+This section describes how this works, using Bob and Alice in the example. 
+
+### Owner
+
+- Bob creates an account on the Aptos blockchain. Now Bob has the `OwnerCapability` resource.
+- Bob can assign his account’s operator address to the account of Alice, a trusted node operator, to appoint Alice as a validator.
+
+As an owner:
+
+- Bob owns the funds that will be used for staking.
+- Only Bob can add or remove funds.
+- Only Bob can extend or renew the lockup period.
+- Bob can change the node operator Alice to some other node operator anytime Bob wishes to do so.
```

Comment:
@movekevin shall we add "The staking reward will be deposit into Bob's account"?